--- /usr/local/lib/python2.7/dist-packages/instapaperlib/instapaperlib.py       2016-01-09 21:39:51.726889676 +0100
+++ /tmp/instapaperlib.py        2016-01-09 22:14:12.958946872 +0100
@@ -150,7 +150,10 @@
         try:
             request = urllib2.Request(url, headerdata)
             response = urllib2.urlopen(request)
-            status = response.read()
+            if 'jsonp' in params:
+                status = response.read()
+            else:
+                status = response.getcode()
             info = response.info()
             try:
                 headers['location'] = info['Content-Location']
